python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -277,12 +277,13 @@ def in_subgraph(self, nodes: torch.Tensor) -> torch.ScriptObject:
         # TODO: change the result to 'SampledSubgraphImpl'.
         return self._c_csc_graph.in_subgraph(nodes)
 
-    def _convert_to_sampled_subgraph(
+    def _generate_sampled_subgraph_node_pairs_and_eids(
```

Comment:
Keep `_convert `in name for consistence with its counterpart  `_convert_to_sampled_subgraph`